speechbrain/lobes/models/dual_path.py

```diff
@@ -280,7 +286,7 @@ class IdentityBlock:
     def _init__(self, **kwargs):
```

Comment:
Suggestion: derive IdentityBlock from `nn.Identity`, make `super().__init__()` call in `__init__`, otherwise the module object won't have needed fields, drop the forward method

Reply:
I replace `IdentityBlock` by `torch.nn.Identity`